### PR TITLE
feat: chaos-dns-server supports global registry

### PR DESCRIPTION
**Pull request by Gallardot**
## Related changes

- [ ] This change also requires further updates to the [website](https://github.com/chaos-mesh/website) (e.g. docs)
- [ ] This change also requires further updates to the `UI interface`

## Cherry-pick to release branches (optional)

> This PR should be cherry-picked to the following release branches:

- [ ] release-2.6
- [ ] release-2.5

### CHANGELOG

> Must include at least one of them.

- [x] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

### Tests

> Must include at least one of them.

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test

### Side effects

- [ ] **Breaking backward compatibility**

## DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: helm/chaos-mesh/values.yaml [hunk 1]
```diff
@@ -410,7 +410,13 @@ dnsServer:
   # Name of serviceaccount for chaos-dns-server.
   serviceAccount: chaos-dns-server
   # Image of DNS Server
-  image: ghcr.io/chaos-mesh/chaos-coredns:v0.2.6
+  image:
+    # override global registry, empty value means using the global images.registry
+    registry: ""
```
Comment (STRRL) [inline helm/chaos-mesh/values.yaml:415]: use ` ghcr.io` for this field?

also update the docs please 

❤️ 

rest LGTM

Comment (Gallardot) [reply helm/chaos-mesh/values.yaml:415]: > use ` ghcr.io` for this field?

@STRRL 
Yes, if there are different configurations, you can modify this field, but empty value means using the global `images.registry`. 

I will update the documentation later.

**Comment by STRRL** [comment]
Hi @Gallardot , we noticed that this PR would introduce some conflicts with this PR. https://github.com/chaos-mesh/chaos-mesh/pull/4091

**Comment by Gallardot** [comment]
> Hi @Gallardot , we noticed that this PR would introduce some conflicts with this PR. #4091

@STRRL 
#4091  LGTM. 

Let's make this PR a supplement to the documentation update.

**Review by g1eny0ung — APPROVED** [review]

**Pull request merged**